build.gradle

```diff
@@ -201,11 +201,11 @@ task installGitHooks(type: Exec) {
 }
 
 
-task deleteOpenAPIDocs {
-//    delete fileTree(dir: "$rootDir/docs/api_reference")
+task cleanAPIDocs(type: Delete) {
+    delete "$rootDir/docs/api_reference"
 }
 
-task generateAPIDocs(/*dependsOn: [deleteOpenAPIDocs],*/ type: GenerateTask) {
+task generateAPIDocs(dependsOn: [cleanAPIDocs], type: GenerateTask) {
```

Comment:
We can add description and group for these task so that it will be visible in `./gradlew task` command. (Ex - we are currently adding it for "installGitHooks" task)

https://docs.gradle.org/current/userguide/command_line_interface.html#sec:listing_tasks